src/rapids_dependency_file_generator/cli.py

```diff
@@ -39,7 +39,7 @@ def validate_args(argv):
     dependent_arg_keys = ["file_key", "output", "matrix"]
     dependent_arg_values = []
     for key in dependent_arg_keys:
-        dependent_arg_values.append(getattr(args, key))
+        dependent_arg_values.append(getattr(args, key) is None)
```

Comment:
I can't make a suggestion here but we should use a list comprehension instead. Also `getattr(...) is None` is the same as `hasattr(...)`, I believe.
```python
dependent_arg_values = [hasattr(args, key) for key in dependent_arg_keys]
```

Reply:
I can change this to a list comprehension. But for the sake of preserving the succinctness of the next line, `if any(dependent_arg_values) and not all(dependent_arg_values):`, I _think_ we need to keep the `is None` logic here to differentiate between an empty string value for `matrix` (e.g. `""`) and it actually being `None`.

If there's a better way to refactor this logic, I'm open to it. I just can't think of any off the top of my head.

Reply:
Okay. That's fair. Then maybe:

```python
dependent_arg_values = [getattr(args, key) is None for key in dependent_arg_keys]
```